src/locales/en/pages.yml

```diff
@@ -49,3 +49,24 @@ prove-identity-another-way:
   h1: Prove your identity another way
   content: You may be able to prove your identity another way with GOV.UK One Login. Or you can go back to the service you need to use and check if there are other ways to access it.
   warning: You cannot return to this screen if you look for another way to prove your identity.
+self-assessment-router:
+  title: What type of Self Assessment tax return did you send for the tax year {{ yearRange }}?
+  content:
+    - This is so we can ask you questions about the right Self Assessment tax return.
+    - You’ll only have sent a short tax return (SA200) if HMRC told you to.
+pensions-benefits-self-assessment:
+  title: Enter pensions and benefits amounts from your Self Assessment tax return
+  h1: Enter pensions and benefits amounts from your Self Assessment tax return
+  content:
+    - <div class="govuk-inset-text"><p>To answer this question, use your Self Assessment tax return for either of these tax years:</p><ul class="govuk-list govuk-list--bullet"><li>2022 to 2023</li><li>2022 to 2023</li></ul></div>
```

Comment:
The tax years will need to be dynamic based on what we get back in the question info object

Reply:
Cool, let's just assume it's a single tax year range for now (as there's a separate ticket to handle multiple)